cli/patch.go

```diff
@@ -501,9 +502,17 @@ func (lc *ListCommand) listDistros() error {
 	if err != nil {
 		return err
 	}
-	fmt.Println(len(distros), "distros:")
+
+	spawnableDistros := []*distro.Distro{}
 	for _, distro := range distros {
-		fmt.Println(distro.Id)
+		if distro.SpawnAllowed {
+			spawnableDistros = append(spawnableDistros, &distro)
+		}
+	}
+
+	fmt.Println(len(spawnableDistros), "distros:")
+	for _, distro := range spawnableDistros {
+		fmt.Println((*distro).Id)
```

Comment:
why not build a slice of strings, and then do a single print statement that does a strings.Join(distros, "\n")?